naughts_crosses.py

```diff
@@ -30,6 +30,30 @@ def prepareBoard():
 
     return board
 
+def drawBoard():
+    global board
+    global COLUMNS
+    global ROWS
+
+    print('| 1 | 2 | 3 | 4 | 5 | 6 |')
+    print('--------------------------------')
+
+    for row in range(ROWS):
+        rowHeight = ROWS - row
+        currentRow = '|'
+        for col in range(1, COLUMNS + 1):
+            if len(board[col]) < rowHeight:
+                currentRow += '   |'
+            elif board[col][rowHeight - 1] == 'O':
```

Comment:
Actually I'm not sure if that would work, seeing as the board gets drawn after each player's go, and the board should be independant from the player who's turn it is (I think)